### DISCUSSION TITLE
Be alerted via Activity section when someone adds an emoji and reacts to your posts

### DISCUSSION THREAD

**Discussion by srkgupta** [mattermost/community#89] [Enhancements]
Often times when someone reacts to your post by adding an emoji, you do not know until you visit and see the posts. 

Just like how we have `Threads` as a separate section, we can add another section called `Activity` which would list all the activities to my posts. For example `Aaron reacted to your post <XYZ>`

![Screenshot 2021-06-24 at 7 29 21 PM](https://user-images.githubusercontent.com/1429138/123276242-9f1cc800-d522-11eb-9242-4c78c0656b62.png)

**Comment by chetanyakan** [comment]
Can be implemented as a plugin.
A bot could DM you about replies to your posts or posts you are interested in.

**Reply by srkgupta** [reply]
No, this is not about replies to posts. This is about notifying when someone adds an emoji and reacts to your posts

**Comment by esethna** [comment]
+1, Related ticket: https://mattermost.atlassian.net/browse/MM-26915
